cmd/encryption-v1.go

```diff
@@ -1157,3 +1159,119 @@ func DecryptObjectInfo(info *ObjectInfo, headers http.Header) (encrypted bool, e
 	}
 	return
 }
+
+// The customer key in the header is used by the gateway for encryption in the case of
+// s3 gateway double encryption. A new client key is derived from the customer provided
+// key to be sent to the s3 backend for encryption at the backend.
+func deriveClientKey(clientKey [32]byte, bucket, object string) ([32]byte, error) {
```

Comment:
The error in the function signature is not used. It should be removed.

Reply:
thanks,fixed this